### PR TITLE
fix: add default activeid value tabs

### LINKED ISSUE #6455: feat: add default value for `activeid` to `Tabs`
# 🙋 Feature Request
When `Tabs` does not have the activeid attribute explicitly set by a developer, the active indicator jumps beyond the next tab that is clicked on before settling into the correct position. By setting the activeid this behavior does not happen.

## 🤔 Expected Behavior
If the active indicator is enabled, the animation for selecting tabs should function smoothly.

## 😯 Current Behavior
The active indicator jumps beyond the first tab selected by a user before settling on the selected tab if the activeid is not set.

## 💁 Possible Solution
Since `Tabs` assigns ids to its child tab elements in the event that page authors don't provide ids to the tabs, it should probably also set the `activeid` to the first non-disabled tab.

## 💻 Examples
This behavior can be seen in both the now archived `fast-components` package https://explore.fast.design/components/fast-tabs and the vNext storybook example.

### PR DESCRIPTION
**Pull request by chrisdholt**
# Pull Request

## 📖 Description
This sets the activeId when finding and setting the active tab during `setTabs()` which is used during setup and updates.

### 🎫 Issues
I believe this closes #6455 (@kingoftac to confirm?)

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

**Review by eljefe223 — APPROVED** [review]

**Review by KingOfTac — APPROVED** [review]
@chrisdholt I must have missed this notification. This looks like it fixes the issue.

File: change/@microsoft-fast-foundation-e71f6ab0-358d-419f-a04e-112f06cf995e.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "set activeid when setting active tab",
+  "packageName": "@microsoft/fast-foundation",
+  "email": "chhol@microsoft.com",
+  "dependentChangeType": "patch"
```
Comment (chrisdholt) [inline change/@microsoft-fast-foundation-e71f6ab0-358d-419f-a04e-112f06cf995e.json:6]:
```suggestion
  "dependentChangeType": "patch"
```

**Comment by Narvalex** [comment]
Is there any ETA for this fix?

**Comment by KingOfTac** [comment]
@chrisdholt I think this can be merged. Now that the active indicator has been removed it won't have much value in the latest version of foundation, however we might want consider back porting to the legacy version.

**Pull request merged**